src/components/buttons/ButtonGroup.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import { ReactNode } from 'react';
+
+export enum ButtonGroupAlign {
+  Left = 'left',
+  Right = 'right',
+}
+
+export interface ButtonGroupProps {
+  align?: ButtonGroupAlign;
+  children: ReactNode;
+  isCombined?: boolean;
+}
+
+export function ButtonGroup({
```

Comment:
Let's forward ref and forward the remaining props too

Reply:
where should the ref go? on the container itself?

Reply:
yeah, we don't have a better alternative anyway

Reply:
what remaining props could there be in this case?

Reply:
I'm wondering what the use case would be for the forwarded ref?

Reply:
Making sure our ButtonGroup is compatible with our other Echoes components that rely on props and ref forwarding.

Reply:
I don't see a use case right now. I just think it's good practice to make sure all our exported components can have a ref attached to them.
We might have usages where we need to know the size or positioning of the group.

Reply:
as discussed on Slack, as there's no use case for now and it's just a "structural" component with no inherent behavior, and it's not used through Redux/Mantine, let's keep it simple.